website/docs/reference/runtimes/random/index.mdx

```diff
@@ -1 +1,56 @@
 # Random
+
+## Random runtime
+The Random runtime allows you to produce *structured* datas randomly.
+
+One use case is to use this runtime as way of testing, for example you can rapidly draft a small backend that produces structured data for your frontend application.
+
+The `seed` parameter ensures repeatability if set.
+
+import TGExample from "@site/src/components/TGExample";
+
+
+<TGExample
+    typegraph="random"
+    python={require("../../../../../examples/typegraphs/roadmap-random.py")}
+    tyepescript={require("!!code-loader!../../../../../examples/typegraphs/roadmap-random.ts")}
+    query={require("./random.graphql")}
+/>
+
+
+Another use case is to inject random values to a materializer input
+<TGExample
+    typegraph="random-field"
+    python={require("../../../../../examples/typegraphs/random-field.py")}
+    query={require("./random-field.graphql")}
+/>
+
+## Generators
+Here is a list of some standard generators that you can attach to your type, if *unspecifed*, it will default on generating any values that the associated type can hold.
+- name
+- age
+- address
+- postcode
+- country
+- email
+- uuid
+- ..etc
```

Comment:
I think we should be exhaustive as we are in the reference section, or at least add a link to where they can find an exhaustive list.